doc/inferring_metrics.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p> </p><p> </p><p>Typo: Morden -&gt; Modern</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgcm/xgcm/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Oops, it is fixed.